runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/AbstractWindmillStream.java

```diff
@@ -133,13 +130,13 @@ private static long debugDuration(long nowMs, long startMs) {
   protected abstract boolean hasPendingRequests();
 
   /**
-   * Called when the stream is throttled due to resource exhausted errors. Will be called for each
-   * resource exhausted error not just the first. onResponse() must stop throttling on receipt of
-   * the first good message.
+   * Called when the client side stream is throttled due to resource exhausted errors. Will be
+   * called for each resource exhausted error not just the first. onResponse() must stop throttling
+   * on receipt of the first good message.
    */
   protected abstract void startThrottleTimer();
 
-  private StreamObserver<RequestT> requestObserver() {
+  protected StreamObserver<RequestT> requestObserver() {
```

Comment:
can this remain private? don't see new call site

Reply:
done.